docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -39,6 +39,586 @@ For more information about versions, build numbers, and Office Online Server, se
 - [Where you can find the version and build number for an Office 365 client application](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
 - [Office Online Server overview](https://docs.microsoft.com/officeonlineserver/office-online-server-overview)
 
+## Excel JavaScript preview API set
+
+New Excel JavaScript APIs are first introduced in "preview" and later become part of a specific, numbered requirement set after sufficient testing occurs and user feedback is acquired. The following table lists the APIs currently available in preview. To provide feedback about a preview API, please use the feedback mechanism at the end of the web page where the API is documented.
+
+> [!NOTE]
+> Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
+>
+> To use preview APIs, you must reference the **beta** library on the CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js and you may also need to join the Office Insider program to get a sufficiently recent Office build.
+
+Included in this release are more than 400 new APIs for Excel. The first table provides a concise summary of the APIs, while the subsequent tables gives a detailed list. Please try the new features and share your feedback with us.
+
+| Feature Area | Description | Relevant Objects/Articles |
+| --- | --- | --- |
```

Comment:
Not sure whether this will affect how the table's being rendered (it looks okay now), but should this syntax be updated to specify "left-aligned"?

`|:---|:---|:---|`